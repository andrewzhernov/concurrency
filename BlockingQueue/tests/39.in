100 1 3 100
