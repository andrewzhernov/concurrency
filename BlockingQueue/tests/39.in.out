Testing with parameters: queue_capacity=100, n_readers=1, n_writers=3, n_items=100
Preparing writers ... 
Preparing readers ... 
Starting test ... 
Done
Shutting queue ... 
Done
Shutted queue ... 
Done
Joining writers ... 
Done
Joining readers ... 
Done
OK
Testing with parameters: queue_capacity=100, n_readers=1, n_writers=3, n_items=100
Preparing writers ... 
Preparing readers ... 
Starting test ... 
Done
Shutting queue ... 
Done
Shutted queue ... 
Done
Joining writers ... 
Done
Joining readers ... 
Done
OK
Testing with parameters: queue_capacity=100, n_readers=1, n_writers=3, n_items=100
Preparing writers ... 
Preparing readers ... 
Starting test ... 
Done
Shutting queue ... 
Done
Shutted queue ... 
Done
Joining writers ... 
Done
Joining readers ... 
Done
OK
